guides/material-3.md

```diff
@@ -410,20 +403,19 @@ We recommend _not_ relying on the `color="primary"`, `color="accent"`, or `color
 that are offered by a number of Angular Material components for M2 themes. However, if you want to
 quickly update to M3 and are willing to accept the extra CSS generated for these variants, you can
 enable backwards compatibility styles that restore the behavior of this API. Call the
-`color-variants-back-compat` mixin from `@angular/material-experimental` with the M3 theme you want
+`color-variants-backwards-compatibilityt` mixin from `@angular/material` with the M3 theme you want
```

Comment:
Theres a small typo in this file, it should be `color-variants-backwards-compatibility`. Theres an extra t at the end